packages/fulfilment-job/src/__tests__/config.spec.js

```diff
@@ -0,0 +1,48 @@
+import AwsMock from 'aws-sdk'
+import config from '../config.js'
+
+describe('config', () => {
+  beforeAll(async () => {
+    AwsMock.SecretsManager.__setResponse('getSecretValue', {
+      SecretString: 'test-ssh-key'
+    })
+    process.env.FULFILMENT_FILE_SIZE = 1234
+    process.env.FULFILMENT_FTP_HOST = 'test-host'
+    process.env.FULFILMENT_FTP_PORT = 2222
+    process.env.FULFILMENT_FTP_PATH = '/remote/share'
+    process.env.FULFILMENT_FTP_USERNAME = 'test-user'
+    process.env.FULFILMENT_FTP_KEY_SECRET_ID = 'test-secret-id'
+    process.env.FULFILMENT_S3_BUCKET = 'test-bucket'
+    await config.initialise()
+  })
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  describe('file', () => {
+    it('provides properties relating to the fulfilment file', async () => {
```

Comment:
don't think this needs to be async - similarly for two of the below functions that don't actually make any await calls

Reply:
You're quite right, it doesn't - I just tend to use this a pattern everywhere with jest.